src/backend/assets/js/session.js

```diff
@@ -1,20 +1,21 @@
 $(document).ready(function() {
+  // name is variable defined in the script in the session template file
+  var eventName = name;
```

Comment:
It is accessible. There is some trick involved here. For individual session handlebar files, we have a separate external js file `session.js`. The name of the event is contained in the `jsonData` object passed to the `handlebars` file. Since the javascript file is externally linked to it, we can't directly use the `jsonData` object passed to the session handlebar file. So, we do something like this in the session handlebar file

```
<script>                                                                                                                       
      var name = "{{eventurls.name}}";                                                                                             
      name = name.substring(name.lastIndexOf('/') + 1);                                                                            </script> 
<script src="../js/session.min.js" type="text/javascript"></script>
```
By doing this, the variable `name` storing the name of the event get accessible to the external `session.js` file and we can use it there.